doc/api/v8.md

```diff
@@ -87,6 +87,24 @@ The value returned is an array of objects containing the following properties:
 ]
 ```
 
+## v8.getHeapdump()
+<!-- YAML
+added: REPACEME
+-->
+
+* Returns: {stream.Readable} A Readable Stream containing the heapdump
+
+Generates a snapshot of the current V8 heap and returns a Readable
+Stream that may be used to read the JSON serialized representation.
```

Comment:
Is it necessary to inform the users that the string is in JSON, if we already tell them that the schema is unspecified? That’s entirely up to v8 and DevTools and will not necessarily hold true forever? The file extension accepted and produced by the DevTools is not .json either, so that’s just leaked implementation details?

Reply:
I prefer it. It at least gives them *some* information about what to expect and gives them the option to try to do *something* with it.